.gitignore

```diff
@@ -0,0 +1,2 @@
+/test.md
+/.github
```

Comment:
- [ ] Kindly be noticed that the .gitignore file listing `test.md` file and `node_modules` directory.

Reply:
Thanks for the feedback, so I should add only `test.md` and `node_modules` to my .gitignore file?

Reply:
That's right.

Reply:
Thanks